cli/src/pcluster/aws/logs.py

```diff
@@ -19,6 +21,14 @@ class LogsClient(Boto3Client):
     def __init__(self):
         super().__init__("logs")
 
+    def log_group_exists(self, log_group_name):
+        """Delete log group by given log group name."""
+        try:
+            self._client.describe_log_streams(logGroupName=log_group_name, limit=1)
```

Comment:
I chose describe_log_streams because it accept a log group name, instead the describe_log_groups only accepts log group name **prefix** so it can return wrong results.

BTW, changed, by reusing the existing describe_log_group method that was already searching for the exact log group name.